tutorial.md

```diff
@@ -1,5 +1,1555 @@
-# How to use my script
+# Tutorial: how to use `vendor_data.py`
 
-In this notebook you will learn how to use the functions and classes defined in `script.py`.
+## Using `vendor_data.py` as a module
 
-This will become a great tutorial.
+This interactive notebook shows how to use the class and the different functions in the script `vendor_data.py`.
+
+To use the script in this code, we can import the script as a module or run it directly.
+We'll begin with importing the code as module:
+
+
+```python
+import sys
+sys.path.append ('vendor_data.py') #this is the relative path to the script. If the script is not directly next to the notebook, it could be better to copy the full path of the notebook.
```

Comment:
no, you don't add the path TO the script, but to the folder where the script lives